package.json

```diff
@@ -37,7 +37,7 @@
     "@visx/shape": "^2.4.0",
     "@visx/tooltip": "^2.8.0",
     "allsettled-polyfill": "^1.0.4",
-    "axios": "^0.24.0",
+    "axios": "0.26.1",
```

Comment:
Do we need to lock this?

Reply:
Not necessarily. I had used yarn to upgrade the package.

Reply:
Though if someone asked me if there are any packages we do need to lock, axios is at the top of the list 😂